app/[...slug]/page.tsx

```diff
@@ -1,9 +1,21 @@
+import { Metadata } from 'next';
 import { getStoryblokApi, ISbStoriesParams, StoryblokClient } from '@storyblok/react/rsc';
 import StoryblokStory from '@storyblok/react/story';
 import { resolveRelations } from '@/utilities/resolveRelations';
+import { getPageMetaData } from '@/utilities/getPageMetadata';
 
 const activeEnv = process.env.NODE_ENV || 'development';
 
+export async function generateMetadata({ params }): Promise<Metadata> {
+  const { data } = await getStoryData(params);
+  const blok = data.story.content;
+
+  let slug: string = params.slug ? params.slug.join('/') : '';
+  const meta = getPageMetaData({ blok, slug });
+
+  return meta;
+}
+
```

Comment:
How we add metadata in nextjs 13 app approach. We need dynamic metadata so this is our only option
https://nextjs.org/docs/app/building-your-application/optimizing/metadata#dynamic-metadata